### PR TITLE
qa/tasks/mgr/dashboard: identify cluster in telemetry test

### PR DESCRIPTION
**Pull request by liewegas**
Set org to 'ceph-qa'.

### CODE REVIEW

File: qa/tasks/mgr/dashboard/test_telemetry.py [hunk 1]
```diff
@@ -11,6 +11,15 @@ def setUpClass(cls):
         data = cls._get('/api/mgr/module/telemetry')
         cls.pre_enabled_status = data['enabled']
 
+        # identify ourselves so we can filter these reports out on the server side
+        self._put(
```
Comment (yaarith) [inline qa/tasks/mgr/dashboard/test_telemetry.py:15]: no `self` in this context

**Comment by liewegas** [comment]
jenkins test api

**Comment by liewegas** [comment]
retest this please

**Review by yaarith — APPROVED** [review]

**Review by epuertat — APPROVED** [review]

**Comment by yaarith** [comment]
`ident` channel check was missing when compiling reports:
https://github.com/ceph/ceph/pull/39538

**Review by avanthakkar — APPROVED** [review]

**Pull request merged**